go-controller/pkg/clustermanager/secondary_network_cluster_manager.go

```diff
@@ -0,0 +1,145 @@
+package clustermanager
+
+import (
+	"github.com/containernetworking/cni/pkg/types"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog/v2"
+
+	ovncnitypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/cni/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	nad "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/network-attach-def-controller"
+	ovntypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+// secondaryNetworkClusterManager object manages the multi net-attach-def controllers.
+// It implements networkAttachDefController.NetworkControllerManager and can be used
+// by NetAttachDefinitionController to add and delete NADs.
+type secondaryNetworkClusterManager struct {
+	// net-attach-def controller handle net-attach-def and create/delete network controllers
+	nadController *nad.NetAttachDefinitionController
+	client        clientset.Interface
+	ovnClient     *util.OVNClusterManagerClientset
+	kube          kube.Interface
```

Comment:
It look like neither `client` or `kube`a re used for anything, could they be removed?